### PR TITLE
SMT-COMP 2020: Enable --fp-exp for new FP logics.

### PR DESCRIPTION
**Pull request by aniemetz**

### CODE REVIEW

**Comment by martin-cs** [comment]
Do we know what formats are being used in the competition?

File: contrib/competitions/smt-comp/run-script-smtcomp-current [hunk 1]
```diff
@@ -104,11 +104,11 @@ BV)
   # finish 10min
   finishwith --full-saturate-quant --no-cegqi-innermost --global-negate
   ;;
-LIA|LRA|NIA|NRA)
+LIA|LRA|NIA|NRA|UFFPDTLIRA|UFFPDTNIRA)
```
Comment (ajreynol) [inline contrib/competitions/smt-comp/run-script-smtcomp-current:107]: Add these two logics to line 88 instead.

File: contrib/competitions/smt-comp/run-script-smtcomp-current [hunk 2]
```diff
@@ -104,11 +104,11 @@ BV)
...
   trywith 30 --full-saturate-quant --nl-ext-tplanes
   trywith 300 --full-saturate-quant --no-cegqi-innermost
   trywith 300 --full-saturate-quant --cegqi-nested-qe
-  finishwith --full-saturate-quant --cegqi-nested-qe --decision=internal
+  finishwith --full-saturate-quant --cegqi-nested-qe --decision=internal --fp-exp
```
Comment (ajreynol) [inline contrib/competitions/smt-comp/run-script-smtcomp-current:111]: Revert this after moving.

**Comment by aniemetz** [comment]
> Do we know what formats are being used in the competition?

nope

**Pull request merged**